config/object_labels/recipe_tea.yaml

```diff
@@ -0,0 +1,58 @@
+version: "1"
+title: "Tea"
+# Note: ids 5-20 are shared with coffee
```

Comment:
Does this mean that ids 5-20 in this file and the coffee files should match? They look different

Reply:
Just updated the ids across all the files